tests/blackbox/stratis_cli_cert.py

```diff
@@ -113,6 +115,29 @@ def setUp(self):
         time.sleep(1)
         exec_command(["udevadm", "settle"])
 
+    def test_permissions(self, command_line, permissions, exp_stdout_empty):
+        """
+        Test running CLI commands with and without root permissions.
+        :param list command_line: The arguments needed to execute the Stratis command being tested
+        :type command_line: List of str
+        :param bool permissions: True if the Stratis command needs root permissions to succeed,
+                                    otherwise False.
+        :param bool exp_stdout_empty: True if stdout is expected to be empty
+                                        when Stratis command succeeds.
+        """
+
+        os.seteuid(_NON_ROOT)
+        try:
```

Comment:
This arrangement seems fine to me, and I would expect it to behave correctly. This is so simple, actually, that there does not seem much obvious benefit to using the multiprocessing module. I feel we should go ahead with the design as is, and file an issue for the multiprocessing modification, to be pursued later.